great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -338,13 +338,18 @@ def __init__(self, runtime_environment: Optional[dict] = None) -> None:
         self._attach_fluent_config_datasources_and_build_data_connectors(
             self.fluent_config
         )
-        oss_id = self._get_oss_id()
         self._anonymizer = Anonymizer(salt=self._data_context_id)
+
+        # TODO: Need to determine V1 mechanism for usage stats opt-out
```

Comment:
V1 analytics is controlled by env variable `GX_ANALYTICS_ENABLED`. Or does this comment mean something else? For the current usage stats, we should always have that disabled and work on removing it completely.

Reply:
Ah okay I didn't know about this env var

Reply:
Correct, if that env variable is disabled, `submit` calls will be a no-op. See [this](https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/analytics/client.py#L59).

Currently, it's just the environment variable, but @jcampbell did raise a great point that we should probably make this env var OR config. Most other OSS packages we looked at do allow for disabling via config.